CMakeLists.txt

```diff
@@ -126,8 +123,6 @@ set(BEEROCKS_LOG_SYSLOG_ENABLED     "false")
 # Platform specific flags
 if (TARGET_PLATFORM STREQUAL "openwrt")
     if (TARGET_PLATFORM_TYPE STREQUAL "ugw")
-        set(BEEROCKS_WLAN1_STEERING_VAPS "wlan0.1")
```

Comment:
Same here - squash to a single commit (c40bb11b7ed1f7d477a836de355f4ce8453c59c7)
Commit name can be `tree-wide: remove steer on vaps from conf files`

Reply:
ok